### ISSUE TITLE
GitAhead fork todo

### ISSUE THREAD

**Issue by JulianGro** [Murmele/Gittyup#2]
I thought it would help to have a little todo list for things that eventually need to be done for the gitahead for.

- [x] Fix up the master branch
- [x] Decide on a new name (obviously we could use a boring placeholder)
  On on our Matrix channel, we decided to use the name *Gittyup* for now
- [x] Update README.md to inform people about the situation (why there is a fork)
  WIP see https://github.com/Murmele/gitahead/pull/3
- [x] Come up with a new logo (obviously we could use a boring placeholder)
- [x] Replace GitAhead logo with new logo
- [x] Further update README.md to get rid of all the original GitAhead links, email-addresses, etc.
  WIP see https://github.com/Murmele/gitahead/pull/3
- [x] Figure out how to do releases (maybe we can split the burden by platform (I have been taking care of the Linux AppImages for Vircadia releases, and would be up for doing that here as well.))

**Comment by Murmele** [comment]
Issues in my master branch:
- I broke somehow single line staging synchronized with the Treewidget and the filewidget
- an issue when scrolling through all files sometimes happens that application crashes

Missing features:
- Working with submodules

**Comment by mmahmoudian** [comment]
I propose the following as well:

- [ ] why you have created a new fork
- [ ] why this name
- [ ] create a website (it's good for branding)
- [ ] create a Matrix/Slack for general purpose chat as you probably need some extra help
- [ ] Adding a screenshot of the application to the README

P.s: I'm one of the guys behind [Flameshot](https://github.com/flameshot-org/flameshot/) and I have done these for that project about a year and a half ago, so let me know if I can also assist you in this project :)

**Comment by Murmele** [comment]
Hi @mmahmoudian,

I created that fork, because I added a lot stuff and it was never merged because of the lack of maintaining of the other project.

There is already a matrix channel: https://matrix.to/#/#Gittyup:matrix.org
There was also decided for that name.

Adding a screenshot is a good idea. At the moment we have only the flatpak build with its preview:https://flathub.org/apps/details/com.github.Murmele.Gittyup

A website would be cool, but I don't have the rwsources yet to do anything.
